Doc/library/json.rst

```diff
@@ -453,6 +467,9 @@ Encoders and Decoders
    .. versionchanged:: 3.4
       Use ``(',', ': ')`` as default if *indent* is not ``None``.
 
+   .. versionchanged:: 3.7
+      Instead of ``(',', ':')`` constant :data:`json.COMPACT` can be used.
```

Comment:
I don't think a `versionchanged` field is necessary as nothing about the function actually changed. I think it would be better to update the paragraph above from "To get the most compact JSON representation,
    you should specify ``(',', ':')`` to eliminate whitespace." to "To get the most compact JSON representation,
    you should specify :attr:`json.COMPACT` to eliminate whitespace.